src/reducers/userReducer.js

```diff
@@ -47,15 +46,14 @@ export type UserReducerAction = {
 
 export const initialState: UserReducerState = {
   data: {
+    username: '',
```

Comment:
Better to use something like `null` here?

Reply:
Good point, but this will require a lot of changes in many screens, I will add as separate PR.